### PR TITLE
dev to main, feat: applications

### PR DESCRIPTION
**Pull request by Assassin-1234**

### CODE REVIEW

**Comment by Assassin-1234** [comment]
@LegItMate can you finish this pr

**Comment by LegItMate** [comment]
> @LegItMate can you finish this pr

Soon:tm:

**Comment by Assassin-1234** [comment]
cool k bye imma procrastinate again

**Comment by Assassin-1234** [comment]
@pro-gamer007 @LegItMate reviews, everything is finished but the readme

File: src/classes/buttonroles.js [hunk 1]
```diff
@@ -16,40 +16,104 @@ class buttonroles {
...
 		this.roles.push({ color: color, label: label, emoji: emoji, role: role });
 		return this;
 	}
 	toJSON() { return { roles: this.roles }; }
 
 	/**
 	 * @param {Message} message - The Discord Message
-	 * @param {MessageEmbed} embed - The Discord Embed
+	 * @param {MessageEmbed} embed - The Discord Embed/Content
 	 * @param {buttonroles} role - The created object using .buttonroles().addrole()
```
Comment (Assassin-1234) [inline src/classes/buttonroles.js:30]: change `content` to `string`. Content itself is a wide guess, is confusing

File: src/classes/dropdownroles.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 const { MessageActionRow, MessageMenuOption, MessageMenu } = require('discord-buttons');
-const { MessageEmbed, Message } = require('discord.js');
```
Comment (Assassin-1234) [inline src/classes/dropdownroles.js:2]: pretty sure Message is required for jsdocs

File: src/classes/main.js [hunk 3]
```diff
@@ -1,100 +1,138 @@
...
+					collector.on('end', async (msgs, reason) => {
+						if (reason == 'ERROR') {
+							return msg.channel.send('That is not a valid answer');
+						}
+						if (reason == 'DONE') {
+							const newdata = await applications.getDataByGuild(menu.message.guild.id);
+							newdata.responses.push(res);
+							await newdata.save();
+							const c = await client.channels.fetch(data.responseChannel, true, false);
+							if (!c) return;
```
Comment (Assassin-1234) [inline src/classes/main.js:114]: make it send a message instead of returning

**Review by Assassin-1234 — COMMENTED** [review]
my review

**Pull request merged**